app/views/candidate_mailer/_get_help.text.erb

```diff
@@ -1,7 +1,7 @@
 # Get help
 
-For advice on becoming a teacher, call Get Into Teaching for free on 0800 389 2500 or chat to an adviser online:
+For advice on becoming a teacher, call <%= service_name.get_into_teaching %> for free on <%= get_into_teaching.tel %> or chat to an adviser online:
 
-https://beta-getintoteaching.education.gov.uk/#talk-to-us
+<%= get_into_teaching.url %><%= get_into_teaching.path_online_chat %>
```

Comment:
I think I'd have a single key for `get_into_teaching.online_chat_url`? If there's duplication across multiple GIT urls then it'll be adjacent in the YAML, and not knowing whether to including a slash when sticking paths on domains makes me anxious